app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[0] == "mv":
+        cmd, source_file, destination_path = parts
```

Comment:
Avoid indexing, use unpacking and default return condition instead
```suggestion
    cmd, source_file, destination_path, *_ = command.split()

    if cmd != "mv":
        return
```